examples/wip/data/cities.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Normally we put data files in separate repo, to avoid bloating this repo.

But this file is so small that you can keep it here. I'd suggest put it directly in the example folder, we want it to be easy to copy an example folder without having dependencies on other folders.

If it is used by more than one example, just copy it into both

Reply:
@1chandu it seems that this file is still in /wip/data/. is there a reason why @ibgreen's comment is not accepted?

Reply:
@shaojingli , as mentioned in my comments, I am going to address all example related comments in following PRs, I had to merge this as per @ibgreen request to unblock him.